js/apps/admin-ui/cypress/e2e/realm_settings_user_profile_enabled.spec.ts

```diff
@@ -156,10 +156,10 @@ describe("User profile tabs", () => {
     it("Removes three validators with the editor", () => {
       getUserProfileTab();
       getJsonEditorTab();
-      userProfileTab.typeJSON(removedThree).saveJSON();
-      masthead.checkNotificationMessage(
```

Comment:
Strange that older messages remain, we do [dismiss notifications automatically](https://github.com/keycloak/keycloak/blob/9caac3814c74be8f4e15dc65b0ada78ff7cb4b13/js/apps/admin-ui/cypress/support/pages/admin-ui/Masthead.ts#L102-L107), so perhaps the previous message was just not being checked at all? I am starting to think we should have a `dismissAllNotifications()` on a `beforeEach()` to prevent this, so that each case starts with a clean slate of notifications.